src/index.html

```diff
@@ -13,7 +13,25 @@
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-
+  <header class="menu">
+    <section class="navigation">
+        <section class="logo">
+            <a href="#">
+              <img src="images/logo.png" alt="Логотип MoYo">
+            </a>
```

Comment:
Follow the indentation rules. Add an additional space of 2 spaces for nested tags/text. 4 - is too much.